README.md

```diff
@@ -22,33 +23,96 @@ The supportet visualizations are:
 - Topoplot series
 - Parallel Coordinates Plot
 
+<details>
+<summary>Click to see example plots</summary>
```

Comment:
we need to update this plot @vladdez do we have a more exciting and more importantly, higher resolution plot?

We also shoudnt hide it, it is a plotting library. Maybe we can shrink it, and not show all plot types?

Reply:
Shrinking it would be nice, I hid it cause it was taking up a looooooot of space